code/sum.py

```diff
@@ -95,17 +94,22 @@ def get_summary(filename="1.mp4", subtitles="1.srt"):
     summary.to_videofile( output, codec="libx264", temp_audiofile="temp.m4a", remove_temp=True, audio_codec="aac")
     return True
 
-def download_video(url):
-    ''' Downloads specified Youtube video
+# download video with subtitles
+def downvdo_subs(subs):
```

Comment:
Kindly rename as `def download_video_srt(url):`

Also, make corresponding changes 👍

Reply:
@AdiChat Consider it done 👍